### PR TITLE
Migrate ide_theme to null safety.

### PR DESCRIPTION
**Pull request by polina-c**

### CODE REVIEW

File: packages/devtools_app/lib/src/config_specific/ide_theme/ide_theme.dart [hunk 1]
```diff
@@ -21,9 +19,9 @@ class IdeTheme {
     this.embed = false,
   });
 
-  final Color backgroundColor;
-  final Color foregroundColor;
-  final double fontSize;
+  final Color? backgroundColor;
+  final Color? foregroundColor;
+  final double? fontSize;
```
Comment (kenzieschmoll) [inline packages/devtools_app/lib/src/config_specific/ide_theme/ide_theme.dart:24]: make font size non-nullable and use unscaledDefaultFontSize as a default where we are currently passing in null for fontSize

File: packages/devtools_app/lib/src/config_specific/ide_theme/ide_theme_web.dart [hunk 2]
```diff
@@ -27,32 +27,32 @@ IdeTheme getIdeTheme() {
...
 
   return overrides;
 }
 
-Color backgroundColor;
-Color foregroundColor;
-double fontSize;
+Color? backgroundColor;
+Color? foregroundColor;
+double? fontSize;
```
Comment (kenzieschmoll) [inline packages/devtools_app/lib/src/config_specific/ide_theme/ide_theme_web.dart:39]: these don't appear to be used anywhere. let's remove.

**Review by kenzieschmoll — APPROVED** [review]
a couple comments then lgtm

File: packages/devtools_app/lib/src/config_specific/ide_theme/ide_theme_web.dart (hunk 2 above)
Comment (polina-c) [reply packages/devtools_app/lib/src/config_specific/ide_theme/ide_theme_web.dart]: When I deleted foregroundColor, I got exception in console, when started DevTools in Chrome and connected to app:
https://gist.github.com/polina-c/f5200a8d693619f33b0bb4e44b38803b

Added TODO.

File: packages/devtools_app/lib/src/config_specific/ide_theme/ide_theme.dart (hunk 1 above)
Comment (polina-c) [reply packages/devtools_app/lib/src/config_specific/ide_theme/ide_theme.dart]: done

**Comment by polina-c** [comment]
tested

**Comment by polina-c** [comment]
tested

**Pull request merged**